### PR TITLE
unit tests added

### LINKED ISSUE #34: Write unit tests
**Describe the feature**
Write unit tests for the methods supported across the `/users` endpoint.

### PR DESCRIPTION
**Pull request by subhangi2731**
fixes #34 Unit tests added

### CODE REVIEW

File: src/app.ts [hunk 1]
```diff
@@ -9,8 +9,8 @@ const app: Application = express()
 const homeRoute = require('./api/routes/home')
 const usersRoute = require('./api/routes/users')
 
-// Connect to database
-mongoose.connect(`mongodb+srv://${process.env.DB_USER}:${process.env.DB_PASS}@cluster0.cogjx.mongodb.net/${process.env.DB_NAME}?retryWrites=true&w=majority`,
+// Connect to database 
+mongoose.connect(`mongodb+srv://${process.env.DB_USER}:${process.env.DB_PASS}@cluster0.hrkum.mongodb.net/${process.env.DB_NAME}?retryWrites=true&w=majority`,
```
Comment (ShauryaAg) [inline src/app.ts:13]: why is this change needed?

File: .env.example [hunk 2]
```diff
@@ -1,3 +0,0 @@
-DB_USER=DB_USER_HERE
```
Comment (singhayushh) [inline .env.example:1]: Why have these variables been deleted?

File: _test_/users.test.js [hunk 3]
```diff
@@ -0,0 +1,13 @@
+const supertest = require('supertest');
+const endpoint = 'https://rest-fetch.herokuapp.com/users'
+
+describe("Testing the users API", () => {
+
+    it("Tests the users route and returns true for status", async () => {
+
+        const response = await supertest(endpoint).get('/');
+        expect(response.status).toBe(200);
```
Comment (singhayushh) [inline _test_/users.test.js:9]: This is just the partial test of the code. You need to check for the response body not just the 200 OK status. And I can see the test for just a single route.

File: package.json [hunk 4]
```diff
@@ -23,7 +24,7 @@
     "body-parser": "^1.19.0",
     "dotenv": "^8.2.0",
     "express": "^4.17.1",
-    "mongoose": "^5.10.9",
+    "mongoose": "^5.12.0",
```
Comment (singhayushh) [inline package.json:27]: Changing the dependency version without consulting is a serious no no. This can lead to a lot of errors if the rest of the code was built on an older version of mongoose and there have been major changes in the future versions, which is the case here.

File: package-lock.json [hunk 5]
```diff
@@ -13,23 +13,62 @@
...
+        "@babel/types": "^7.13.0",
+        "convert-source-map": "^1.7.0",
+        "debug": "^4.1.0",
+        "gensync": "^1.0.0-beta.2",
+        "json5": "^2.1.2",
+        "lodash": "^4.17.19",
+        "semver": "^6.3.0",
+        "source-map": "^0.5.0"
       },
       "dependencies": {
```
Comment (singhayushh) [inline package-lock.json:45]: You commit should not include package-lock.json file

**Review by singhayushh — CHANGES_REQUESTED** [review]
